pom.xml

```diff
@@ -1164,6 +1171,21 @@ limitations under the License.
               </rules>
             </configuration>
           </execution>
+          <execution>
+            <id>ban-unwanted-imports</id>
+            <phase>process-sources</phase>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <RestrictImports>
```

Comment:
Can't this go into the first execution?

Reply:
seems to work as well, so pushed another commit